src/components/settings/SettingsSSOTab/NewSSOConfigCard.jsx

```diff
@@ -49,9 +49,16 @@ const NewSSOConfigCard = ({
     });
   };
 
+  const renderKeyOffIcon = (dataTestId) => (
```

Comment:
This is only ever called with the value `existing-sso-config-card-not-validated-icon`, should it just be a const?

Reply:
good point, for a second I was considering adding others but this should just be hard coded

Reply:
actually, now I have two separate test ID's being passed in